Prerelease metrics/v0.32.1 (#3228)
open-telemetry/opentelemetry-go: CHANGELOG.md

## [Unreleased]

> ## [0.32.1] Revised Metric SDK (Alpha) - 2022-09-22

Reviewer: ```suggestion
## [0.32.1] Metric SDK (Alpha) - 2022-09-22
```

### Changed

> - Instead of dropping metric, the Prometheus exporter will replace any invalid character in metric names with `_`. (#3212)

Reviewer: ```suggestion
- The Prometheus exporter sanitizes OpenTelemetry instrument names when exporting.
   Invalid characters are replaced with `_`. (#3212)
```
